data/projects.js

```diff
@@ -561,5 +561,26 @@ coala should support generating metrics for your code.",
 	  "tags" : ["Information Extraction", "CLI", "CI"],
 	  "markdown" : "enhance_coala_quickstart.md",
 	  "collaborating_projects" : ["coala"]
+	},
+	{
+		"name" : "Newcomer metrics and gamification",
```

Comment:
Statistics about individuals will always create a game, either implicit or explicit, either defined by us, or played without clear definition.
We need to define the way we think the game should be played, and how we will score it.
I do appreciate your point, and have experienced bad gamification often enough to be concerned, but while we are welcoming to newcomers, we also tell them to hike up a steep cliff of learning new process and skills, which eliminates the most likely to cause problems.  I am sure we can tweak the game enough that it hurts those who consume too much of our time.